### PR TITLE
Update __init__.py to Add Support for DeArrow API

### PR DESCRIPTION
**Pull request by bawitdaba**
Added Support for DeArrow API to obtain Crowd Sourced Video Titles.
See https://dearrow.ajay.app/ and https://wiki.sponsor.ajay.app/w/API_Docs/DeArrow for more information on DeArrow which is created by the SponsorBlock author.

### CODE REVIEW

**Comment by ZeroQI** [comment]
Very good idea! Didn't even know it existed...

Would feel better if it was optional though, as an agent option set to disable by default

https://github.com/ZeroQI/YouTube-Agent.bundle/blob/master/Contents/DefaultPrefs.json

```
{ "id":"dearrow",          "label":"DeArrow API crowdsourced title too avoid clickbait
 titles",  "type":"bool", "default":"false"                                     }
```

```
  If Prefs['dearrow']==True: 
    crowd_sourced_title = DeArrow(guid)
    if crowd_sourced_title != '':
      metadata.summary = 'Original Title: ' + 
      metadata.title + '\r\n\r\n' + metadata.summary
      metadata.original_title = metadata.title
      metadata.title = crowd_sourced_title
```

File: Contents/Code/__init__.py [hunk 1]
```diff
@@ -24,6 +25,36 @@ def Dict(var, *arg, **kwarg):  #Avoid TypeError: argument of type 'NoneType' is
...
+  url = '{api_url}/api/branding/{hash}'.format(api_url = api_url, hash = hash[:4])
+
+  try:
+    data_json = {}
+    data_json = JSON.ObjectFromURL(url)
+  except:
+    crowd_sourced_title = ''
+
+  try:
+    crowd_sourced_title = titlecase(Dict(data_json[video_id]['titles'][0], 'title'))
```
Comment (ajayyy) [inline Contents/Code/__init__.py:52]:
```suggestion
    first_title = video_id]['titles'][0]
    crowd_sourced_title = titlecase(Dict(first_title, 'title')) if Dict(first_title, 'votes') > 0 or Dict(first_title, 'locked')
```

Comment (ajayyy) [reply Contents/Code/__init__.py:52]: Eliminates untrusted titles from being used (in the extension these appear in the voting menu only, but not on videos themselves)

Comment (bawitdaba) [reply Contents/Code/__init__.py]: Thank you! I merged your code with the suggested changes from @ZeroQI and refreshed the pull request

**Pull request merged**